app/screens/channel_info/bindings/bindings.tsx

```diff
@@ -21,14 +21,15 @@ type Props = {
     theme: Theme;
     currentChannel: Channel;
     appsEnabled: boolean;
+    intl: typeof intlShape;
     currentTeamId: string;
     actions: {
         doAppCall: (call: AppCallRequest, type: AppCallType, intl: any) => Promise<{data?: AppCallResponse, error?: AppCallResponse}>;
         sendEphemeralPost: (message: any, channelId?: string, parentId?: string) => Promise<ActionResult>;
     }
 }
 
-const Bindings: React.FC<Props> = (props: Props) => {
+const Bindings: React.FC<Props> = injectIntl((props: Props) => {
```

Comment:
If we are using a class component later, I think we can remove the intl injection here and use the context intl in the class component.

Reply:
👋 I'd say keep this approach as the context approach has been deprecated in newer versions of react-intl ;)